docs/release-notes/v11.5.5.md

```diff
@@ -0,0 +1,18 @@
+---
+id: v11.5.5
+title: Version 11.5.5 Release Notes
+sidebar_label: v11.5.5
+image: "img/release-note-link-preview.png"
+---
+
+_Oct 16th, 2023_
+
+## Console
+
+### Customized branding in the console 
+
+From now on it will be possible to set your custom brand in the Console, so that it replaces Mia-Platform's one both within the Console and in the login pages. To enable this customization, please contact your product contact person.
```

Comment:
```suggestion
From now on it will be possible to set your custom brand in the Console, so that it replaces Mia-Platform logo one both within the Console and in the login pages. To enable this customization, please contact your product contact person.

:::info
This feature is only available for self-hosted installations
:::
```